content/resources/tutorials/nipoppy.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Streamlining Neuroimaging Processing with Nipoppy
-linkTitle: Nipoppy
+linkTitle: Higher order process management
```

Comment:
What about "Higher order process management with Nipoppy"

IMO having the tool name in there will be helpful